docs/getting-started/README.md

```diff
@@ -0,0 +1,73 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.9.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Getting Started with Calico
+
+## Networking options
+
+Worked examples are available for demonstrating Calico networking with the 
+following different networking options.
+
+### Docker default networking
+
+This uses Dockers standard networking infrastructure, requiring you to 
+explicitly add a created container into a Calico network.
+
+This is compatible with all Docker versions from 1.6 onwards.
+
+### Docker with libnetwork
+
+Docker's native [libnetwork network driver][libnetwork] is available in the 
+Docker 1.9 release currently underoing development.
```

Comment:
/s/underoing/undergoing, although this sentence might need tweaking to reflect the fact that Docker 1.9 is in code slush.